front/components/data_source/DataSourceSyncChip.tsx

```diff
@@ -74,6 +75,25 @@ export default function ConnectorSyncingChip({
             trigger={<Chip color="warning">Synchronization failed</Chip>}
           />
         );
+      case "webcrawling_error_empty_content":
+        return (
+          <Tooltip
```

Comment:
Yes it works, but the tooltip is too small, I'll follow up after having decided what to put in the tooltip (it could already be small enough).

<img width="501" alt="tooltip_too_small" src="https://github.com/user-attachments/assets/217e8405-b7b7-4262-9424-6910cd4c2872">

Reply:
We've got a tooltip issue here, if we can't read the full text right?

Reply:
yep, fixed it!